big_tests/tests/domain_isolation_SUITE.erl

```diff
@@ -35,10 +35,12 @@ host_types() ->
 %%--------------------------------------------------------------------
 
 init_per_suite(Config) ->
+    instrument_helper:start([{router_stanza_dropped, #{host_type => host_type()}}]),
```

Comment:
Minor, but maybe use the declared events helper for consistency with other test suites?

Reply:
I am declaring the event explicitly to avoid the case when adding a new event to the module would cause tests to fail.